INSTALL.md

```diff
@@ -1,25 +1,7 @@
-# Table of contents
-
-* [Platforms](#platforms)
-* [Get the source code](#get-the-source-code)
-* [Dependencies](#dependencies)
-  * [Windows dependencies](#windows-dependencies)
-  * [Mac OS X dependencies](#mac-os-x-dependencies)
-  * [Linux dependencies](#linux-dependencies)
-* [Compiling](#compiling)
-  * [Windows details](#windows-details)
-  * [Mac OS X details](#mac-os-x-details)
-    * [Issues with Retina displays](#issues-with-retina-displays)
-  * [Linux details](#linux-details)
-* [Using shared third party libraries](#using-shared-third-party-libraries)
-  * [Linux issues](#linux-issues)
-* [Building Skia dependency](#building-skia-dependency)
-  * [Skia on Windows](#skia-on-windows)
-  * [Skia on Mac OS X](#skia-on-mac-os-x)
```

Comment:
Index was a lot useful, please restore it and update it

Reply:
GitHub already provides a table of contents if you click the menu icon in the top-left corner, so I thought we might not need one.

Reply:
I had no idea, thanks. But I still think it's cleaner and more explicit with the index (also, one click less, enjoy laziness)